cvat/apps/engine/task.py

```diff
@@ -592,10 +594,13 @@ def _create_thread(
     slogger.glob.info("create task #{}".format(db_task.id))
 
     job = rq.get_current_job()
+    rq_job_meta = ImportRQMeta.from_job(job)
 
-    def _update_status(msg: str) -> None:
-        job.meta['status'] = msg
-        job.save_meta()
+    def _update_status(rq_job_meta: ImportRQMeta, msg: str) -> None:
```

Comment:
Seems like you could do:

```suggestion
    def update_status(msg: str) -> None:
```